atom/browser/api/atom_api_web_contents.cc

```diff
@@ -771,6 +771,7 @@ void WebContents::AddNewContents(content::WebContents* source,
   }
 
   bool active = disposition != WindowOpenDisposition::NEW_BACKGROUND_TAB;
+  atom::NativeWindow* owner_window = nullptr;
```

Comment:
why is this declared way up here as `nullptr`? It's only used inside the block below